src/lib/utils/bitvector.h

```diff
@@ -1,6 +1,8 @@
 /*
  * An abstraction for an arbitrarily large bitvector that can
```

Comment:
@reneme the code still contains some artefacts for varying the underlying data type that we'd ideally want to get rid of.